jcl/src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -72,7 +80,11 @@ String getMethodDescriptor() {
 	}
 	/*[IF Java11]*/
 	public boolean isFinal() {
-		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+		if (mh instanceof FieldHandle) {
+			return ((FieldHandle)mh).isFinal();
+		}
+		/*[MSG "K0675", "Unexpected MethodHandle instance: {0}"]*/
+		throw new InternalError(com.ibm.oti.util.Msg.getString("K0675", mh)); //$NON-NLS-1$
```

Comment:
That's a good first step but `mh.toString()` only shows the 'MethodHandle(signature)returntype'.  It would be good to include the `mh.getClass()` as well.

Reply:
Added `mh.getClass()` info, now the `InternalError` to be thrown might look like:
```
java.lang.InternalError: Unexpected MethodHandle instance: MethodHandle()int with class java.lang.invoke.StaticFieldGetterHandle
	at java.lang.invoke.MemberName.isFinal(java.base@11-internal/MemberName.java:74)
	at java.lang.invoke.ConstantBootstraps.getStaticFinal(java.base@11-internal/ConstantBootstraps.java:168)
```